cms/grading/scoretypes/abc.py

```diff
@@ -392,25 +382,29 @@ def compute_score(self, submission_result):
 
             testcases = []
             public_testcases = []
-            previous_tc_all_correct = True
+
+            restricted_feedback_idx = None
+            restricted_feedback_outcome = sys.float_info.max
```

Comment:
ultra-nit: I am wondering whether we should do `restricted_feedback_outcome = None`
Then in line 405, change to `if restricted_feedback_outcome is None or outcome < restricted_feedback_outcome`

With this approach, `restricted_feedback_idx` will not be `None` even if all public testcases `outcome` is `sys.float_info.max`.

(Don't forget to remove `import sys` if you want to change this)